### PR TITLE
Move idp config to database

### PR DESCRIPTION
**Pull request by bmanth60**
Hi All,

I have moved the idp configuration to the database. I have tested using the docker containers for single site and multi site.

Please review @ewbdaniel @ewbgray @SamFiltered @tricaalexandru

### CODE REVIEW

File: src/lib/classes/saml_admin.php [hunk 1]
```diff
@@ -183,8 +211,22 @@ public function get_saml_status()
         }
       }
     }
-    
-    if(file_exists(constant('SAMLAUTH_CONF') . '/certs/' . get_current_blog_id() . '/' . get_current_blog_id() . '.cer') && file_exists(constant('SAMLAUTH_CONF') . '/certs/' . get_current_blog_id() . '/' . get_current_blog_id() . '.key'))
+
+    /**
```
Comment (SamFiltered) [inline src/lib/classes/saml_admin.php]: /** should only be used for DocBlocks

File: src/lib/classes/saml_client.php [hunk 2]
```diff
@@ -55,6 +55,17 @@ public function authenticate()
         $username = $attrs[$this->settings->get_attribute('username')][0];
         if(get_user_by('login',$username))
         {
+            /**
```
Comment (SamFiltered) [inline src/lib/classes/saml_client.php]: ditto

**Comment by SamFiltered** [comment]
:shipit:

File: src/lib/classes/saml_admin.php [hunk 3]
```diff
@@ -183,8 +211,22 @@ public function get_saml_status()
...
+     * otherwise check for cert files.
+     */
+    if(
+        (
+            $this->settings->get_public_key()
+            && $this->settings->get_private_key()
+        )
+        ||
+        (
+            file_exists(constant('SAMLAUTH_CONF') . '/certs/' . get_current_blog_id() . '/' . get_current_blog_id() . '.cert')
```
Comment (tricaalexandru) [inline src/lib/classes/saml_admin.php]: Mine is created with '.cer'. Are you sore it's '.cert'?

**Comment by tricaalexandru** [comment]
@bmanth60 is this mergeable?

File: src/lib/classes/saml_admin.php (hunk 3 above)
Comment (bmanth60) [inline src/lib/classes/saml_admin.php]: @tricaalexandru good catch, thanks!

**Comment by bmanth60** [comment]
@tricaalexandru if you give it a :shipit: then it can be merged.

**Pull request merged**